utilities_common/multi_asic.py

```diff
@@ -5,57 +5,29 @@
 
 import click
 
-from sonic_device_util import get_all_namespaces
-from sonic_device_util import get_namespaces
-from sonic_device_util import get_num_npus
-from sonic_device_util import is_multi_npu
-from sonic_device_util import is_bgp_session_internal
-from sonic_device_util import is_port_channel_internal
-from sonic_device_util import is_port_internal
-from swsssdk import ConfigDBConnector
-from swsssdk import SonicDBConfig
-from swsssdk import SonicV2Connector
-from utilities_common.constants import *
+from sonic_py_common.multi_asic_device_info import connect_config_db_for_ns
+from sonic_py_common.multi_asic_device_info import connect_to_all_dbs_for_ns
+from sonic_py_common.multi_asic_device_info import get_all_namespaces
+from sonic_py_common.multi_asic_device_info import get_namespace_list
+from sonic_py_common.multi_asic_device_info import is_multi_asic
+from sonic_py_common.multi_asic_device_info import is_bgp_session_internal
+from sonic_py_common.multi_asic_device_info import is_port_channel_internal
+from sonic_py_common.multi_asic_device_info import is_port_internal
+from utilities_common.constants import DEFAULT_NAMESPACE
+from utilities_common.constants import DISPLAY_ALL
+from utilities_common.constants import DISPLAY_EXTERNAL
+from utilities_common.constants import BGP_NEIGH_OBJ
+from utilities_common.constants import PORT_CHANNEL_OBJ
+from utilities_common.constants import PORT_OBJ
```

Comment:
Is it better to import the entire module (`from utilities_common import constants`)?

Reply:
Changed in the latest commit